### DISCUSSION TITLE
Organization discussion

### DISCUSSION THREAD

**Discussion by yoshinorin** [hexojs/hexo#5059] [Ideas]
@hexojs/core 

Should we use `organization discussion` instead of each repos discussion?

> https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization

I assume the current (each repos) discussions can be transferred into `organization discussion` . If we use it, we can put together all repos QA to there.

So, if we enable it we should do...

1. enable  `organization discussion`
2. current discussions in all repos transfer into `organization discussion` 
3. disable each repos discussion

How do you think?

Thank you :)

**Comment by renbaoshuo** [comment]
👍 Great idea!

**Comment by yoshinorin** [comment]
Just now I enabled org discussions. But, honestly I don't know how should we use this.
I assumed we can transfer from each repositories' issue to org discussions. But, it can't.

> https://github.com/orgs/hexojs/discussions

Do you have a any good idea for use this?
